pkg/acsengine/munge.go

```diff
@@ -0,0 +1,194 @@
+package acsengine
```

Comment:
I disagree with names being "C#-y". The name should represent what the file is doing. It can be something else but I don't think munge is more Go-like either.